client/asserts_test.go

```diff
@@ -25,9 +25,11 @@ import (
 	"net/http"
 	"net/url"
 
+	"gopkg.in/check.v1"
```

Comment:
I think this is a mistake :-) while I don't like the `.` import, when it's used I'd respect it rather than introduce both approaches in the same file...

Reply:
yes this was a copy and paste error, my editor automatically adds needed packages when it can find them